spread.yaml

```diff
@@ -66,8 +96,7 @@ prepare: |
 
```

Comment:
curl is not able to fetch from api.github.com within this runner too (how was it working before?).
See those [logs](https://github.com/canonical/chisel-releases/actions/runs/10522623910/job/29155637853?pr=258).

Reply:
that's a one-off timeout.

Reply:
I repeated the CI run several times before that, and curl wasn't getting a response.

Reply:
y. and I can't reproduce the `curl` error locally. looks like a docker networking issue.

does it happen to you locally when running spread?

Let's get the LXD backend merged and hope this doesn't happen anymore

Reply:
It was just on the CI.
Yeah most probably a temporary networking error.